bitnami/spark/README.md

```diff
@@ -176,6 +176,7 @@ The following tables lists the configurable parameters of the spark chart and th
 | `security.rpc.authenticationEnabled` | Enable the RPC authentication                                           | `false`    |
 | `security.rpc.encryptionEnabled`     | Enable the encryption for RPC                                           | `false`    |
 | `security.storageEncryptionEnabled`  | Enable the encryption of the storage                                    | `false`    |
+| `security.autoGeneratedCertificates` | Generate automatically self-signed TLS certificates                     | `false`    |
 | `security.ssl.enabled`               | Enable the SSL configuration                                            | `false`    |
```

Comment:
Shouldn't this parameter be under the `security.ssl` block? E.g. `security.ssl.autoGenerated`

Reply:
I initially added it at the same level as `security.certificatesSecretName`. 

Checking [Spark documentation](https://spark.apache.org/docs/latest/security.html), I thought it could be because the certificates were used for other Security settings

After having a look at the Spark container logic, it seems they are used to configure `security.ssl.*` settings when `SPARK_SSL_ENABLED` is true:
https://github.com/bitnami/bitnami-docker-spark/blob/742c07b9dc812900670e3687a50a9453ca4a2828/3/debian-10/rootfs/opt/bitnami/scripts/libspark.sh#L227-L243

Reply:
I will then standardize both settings in a backward-compatible way:

`security.autoGeneratedCertificates -> security.ssl.autoGenerated`
`security.certificatesSecretName -> security.ssl.existingSecret` (security.certificatesSecretName will be Deprecated)